wgengine/magicsock/magicsock.go

```diff
@@ -4920,7 +4920,25 @@ func betterAddr(a, b addrLatency) bool {
 			return false
 		}
 	}
-	return a.latency < b.latency
+
+	// If a is higher-latency than b, then it's not better.
+	if a.latency >= b.latency {
```

Comment:
This assumes `a` is the current selected best?
What if this difference is less than 1%?